lib/migration.pm

```diff
@@ -207,4 +209,13 @@ sub set_scc_proxy_url {
     save_screenshot;
 }
 
+# Enable ZYPP_SINGLE_RPMTRANS=1 if the test case has ZYPP_SINGLE_RPMTRANS setting
+# jira feature https://jira.suse.com/browse/SLE-21973
+# zypper single transaction performance enhancement (RFE)
```

Comment:
Please use perl POD or move the comments within the functtion :)

Reply:
done, thanks